### PR TITLE
[homework #5] - CSS Advanced by German Macchia

### PR DESCRIPTION
**Pull request by GermanMacchia**
Hey @VitalikMelnyk!
I think I finished the last homework. 
I added some animations, media for tablet and a button for top of the page.
Please ping me for any feedback or recomendation,
Thanks!

### CODE REVIEW

**Comment by VitalikMelnyk** [comment]
Please, before pushing homework, run the format with Prettier.

File: homeworks/german-macchia/05-css-advanced/styles/mobile.css [hunk 1]
```diff
@@ -0,0 +1,114 @@
+@media (max-width: 600px) {
```
Comment (VitalikMelnyk) [inline homeworks/german-macchia/05-css-advanced/styles/mobile.css:1]: The same as my comments in `style.css`

File: homeworks/german-macchia/05-css-advanced/styles/tablet.css [hunk 2]
```diff
@@ -0,0 +1,140 @@
+
```
Comment (VitalikMelnyk) [inline homeworks/german-macchia/05-css-advanced/styles/tablet.css:1]: The same as my comments in `style.css`

**Comment by VitalikMelnyk** [comment]
By the way,
The best practice is to create more small logic commits, unlike one big commit.
Please, in the next homework create more logic commits.

**Comment by GermanMacchia** [comment]
HI @VitalikMelnyk !
I made the changes in the style files! I think everything it is more tidy and understandable.
I will take note about the commits best practice. It makes lot of sence to commit by feature changes or logic blocks.
Thanks for the advice!
If you have any other feedback, I'll be glad to know.

**Comment by GermanMacchia** [comment]
Hi @VitalikMelnyk !
I just finished your tutorial for responsive css and I think I'll need to make some new changes to this homework! 
I will push the new version soon!
Thanks for all!

**Comment by GermanMacchia** [comment]
Hi @VitalikMelnyk!
I finished fixing my file. I realiced I didn't get the gist about the using the figma mock up. I used  fixed sizes showed on it and after I looked your tutorial I realiced my mistake. Think I made some progress by using relative and responsible variables as values. 
I hope this time I got it! Have a nice day!

**Comment by VitalikMelnyk** [comment]
Hey @GermanMacchia 
Everything is almostt ok.
Just one comment, please review and change, and I will merge your PR.
Thanks.

**Comment by GermanMacchia** [comment]
Hey @VitalikMelnyk,
I change some comments in the style files. I'm not sure if that was the thing I had to resolve. 
I checked everything anyway:

- [ ] BEM
- [ ] Format
- [ ] Comments

Sorry for that!

**Comment by VitalikMelnyk** [comment]
It's about @keyframes @GermanMacchia.
It's telling you to use the web standard [@keyframes] definition. However, in the example you've provided, you're using [@-webkit-keyframes], which is non-standard.

To fix the error, you need to change @-webkit-keyframes to @keyframes. Furthermore, you can find the browser support of the standard @keyframes definition on [Can I Use? @keyframes](https://caniuse.com/?search=%40keyframes)

**Comment by GermanMacchia** [comment]
Hey @VitalikMelnyk !
So I see!! To be honest I tooked animations from [https://animista.net/ ](url)
I can see -webkit- is kind of deprecated as it used to be for turn the code compatible with some browsers. 
I was checking the source [https://caniuse.com/?search=%40keyframes](url) and see that was not necessary either for transform. 
Thanks for remarking me that!

**Comment by VitalikMelnyk** [comment]
> Hey @VitalikMelnyk ! So I see!! To be honest I tooked animations from https://animista.net/ I can see -webkit- is kind of deprecated as it used to be for turn the code compatible with some browsers. I was checking the source https://caniuse.com/ and see that was not necessary either for transform. Thanks for remarking me that!

**Comment by VitalikMelnyk** [comment]
> Hey @VitalikMelnyk ! So I see!! To be honest I tooked animations from https://animista.net/ I can see -webkit- is kind of deprecated as it used to be for turn the code compatible with some browsers. I was checking the source https://caniuse.com/ and see that was not necessary either for transform. Thanks for remarking me that!

Please, check one thing that I comment on above.

**Review by VitalikMelnyk — APPROVED** [review]

**Pull request merged**